src/twosection.jl

```diff
@@ -0,0 +1,85 @@
+"""
+    TwoSectionView{T<:Real} <: AbstractGraph{Int64}
+
+Create a 2-section view of a hypergraph `h`.
+Note this is a view - changes to the original hypergraph will be automatically reflected in the view.
+
+The 2-section view of a hypergraph is suitable for processing with the LightGraphs.jl package.
+Several LightGraphs methods are provided for the compability.
+
+"""
+struct TwoSectionView{T<:Real} <: AbstractGraph{Int}
+    h::Hypergraph{T}
+    function TwoSectionView{T}(h::Hypergraph{T}) where {T<:Real}
```

Comment:
I do not think this constructor is needed

Reply:
Don't we have to create a new type as for the BipartiteView?

Reply:
This constructor will be automatically added without defining it. In `BipartiteView` it is also not needed. We could define a constructor `TwoSectionView(h::Hypergraph{T}) where {T<:Real}` so that user does not have to specify the type parameter in the call.

@pszufe The same for `BipartiteView`.

Reply:
However, I have just reminded myself that this constructor (without parameter) is also auto-generated. So we do not need to define any constructors in the end.

Reply:
> In this way we can use a shorter form of constructing a view

As I have said above - this shorter form is also autogenerated.